src/components/Character/Character.jsx

```diff
@@ -19,34 +22,41 @@ export default function Character(props){
 		name,
 		nickname,
 		actor,
-		body,
-		image = {},
+		body: bodyData,
+		image:{
+			src,
+			mask,
+			description
+		} = {},
 		index,
 		id
 	} = props;
 
-	const {
-		src,
-		mask: maskPath,
-		description
-	} = image;
-
-
-	// inline style to prevent the need for a huge png asset
-	const mask = { clipPath: `url(${maskPath})` };
-
-
-	// next & prev button labels
-	const prevCharacter = characterData[index - 1] || {};
-	const nextCharacter = characterData[index + 1] || {};
+	// wrap indexes if less than 0 or greater than number of characters
+	let nextIndex = index + 1;
+	let prevIndex = index - 1;
+	
+	if (nextIndex === characterData.length) nextIndex = 0;
+	if (prevIndex < 0)                      prevIndex = characterData.length-1;
 
+	const prevCharacter          = characterData[prevIndex];
+	const nextCharacter          = characterData[nextIndex];
 	const { name: prevCharName } = prevCharacter;
 	const { name: nextCharName } = nextCharacter;
 
+	const prevButton = RENDER.button(prevCharName, "prev", prevIndex);
+	const nextButton = RENDER.button(nextCharName, "next", nextIndex);
 
 	// active character checks
 	const hidden = index !== activeIndex;
 
+	const body = RENDER_TEXT.body(bodyData, {
```

Comment:
as above - `RENDER.body` is better, I think, as it shows you're using the `RENDER` interface - this just looks like a weirdly named function or class or something?